### ISSUE TITLE
ImagePicker.openPicker not working on Samsung Galaxy

### ISSUE THREAD

**Issue by musthafatechrish** [ivpusic/react-native-image-crop-picker#1927]
### Version
Tell us which versions you are using: 

- react-native-image-crop-picker v0.39.0
- react-native v0.71.2

### Platform
Tell us to which platform this issue is related

- Android

### Expected behaviour
Should open storage for choosing files from internal / external storage

### Actual behaviour
Not opening the storage

Love react-native-image-crop-picker? Please consider supporting our collective:
👉  https://opencollective.com/react-native-image-crop-picker/donate

**Comment by ejain** [comment]
Is this on Android 13, and does the error state that the "android.permission.READ_MEDIA_IMAGES" permission is missing?

**Comment by hoangtam101** [comment]
I think you miss the line code in AndroidManifest
`<uses-permission android:name="android.permission.READ_MEDIA_IMAGES" />`

I check it still works on Android 13.
Thanks and regards,

**Comment by steve-smith93** [comment]
Yes i have missed above line of code in AndroidManifest. After adding the same the feature is working fine now.

**Comment by Siliconvelly** [comment]
thanks buddy well. it is working for android 13 and api >= 33 devices now.

**Comment by ayoubshah2720** [comment]
Add these permissions and it will work
![image](https://github.com/ivpusic/react-native-image-crop-picker/assets/48887313/a62deb8e-8f4b-4e54-b293-756f6c652b2f)
